sdk/forms/src/main/java/com/klaviyo/forms/KlaviyoWebView.kt

```diff
@@ -0,0 +1,62 @@
+package com.klaviyo.forms
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.graphics.Color
+import android.util.AttributeSet
+import android.webkit.WebView
+import androidx.webkit.WebViewCompat
+import androidx.webkit.WebViewFeature.WEB_MESSAGE_LISTENER
+import androidx.webkit.WebViewFeature.isFeatureSupported
+import com.klaviyo.analytics.DeviceProperties
+import com.klaviyo.core.BuildConfig
+import com.klaviyo.core.Registry
+
+/**
+ * View logic for in-app forms
+ */
+@SuppressLint("SetJavaScriptEnabled")
+internal class KlaviyoWebView : WebView {
+    constructor(
+        context: Context = Registry.config.applicationContext
+    ) : super(context) {
+        settings.userAgentString = DeviceProperties.userAgent
+        settings.javaScriptEnabled = true
+        settings.domStorageEnabled = true
+        setBackgroundColor(Color.TRANSPARENT)
+
+        if (BuildConfig.DEBUG) {
+            setWebContentsDebuggingEnabled(BuildConfig.DEBUG)
+            // Disable webview resources cache when debugging:
+            // settings.cacheMode = WebSettings.LOAD_NO_CACHE
```

Comment:
nit: should we control through some env vars or something instead of just leaving commented out?

Reply:
I had it controlled thru the DEBUG build flag, but depending on what you're doing, sometimes the cache is useful even if debugging. So I just left this as a breadcrumb to remember it's a tool available to us. I am fine with deleting it before release tho